os-autoinst/os-autoinst-distri-opensuse
# Test modules for transactional updates and rebootmgr

Test transactional-updates
  - installs & remove ptf, update, rollback
  - check that system was rebooted and mounted snapshot changed
  - check that expected package version match

Test rebootmgr using instantly & maint-window & etcd-lock strategies

poo#17056 - transactional-update
poo#16266 - rebootmgr

Needles: https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/315

Local runs:
http://dhcp91.suse.cz/tests/4014 - DVD
http://dhcp91.suse.cz/tests/4009 - VMX
